src/content/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations.mdx

```diff
@@ -8,61 +8,126 @@ translate:
-
-Additional <DNT>**API Polling**</DNT> integrations can be enabled on top of the AWS CloudWatch metric streams in order to pull data that's not available as CloudWatch metrics. The following integrations are not replaced by the metric streams:
+New Relic offers several methods to collect performance data about your AWS environment. Once you've connected New Relic with AWS, you get total visibility into AWS telemetry data, letting you closely monitor the health of your AWS stack. 
+
+## Integrate New Relic with your AWS stack [#connect]
+
+To get started, you need to establish a connection between New Relic and your AWS services. Establishing the initial connection integrates New Relic into your AWS stack so you can access additional integrations for full visibility into your AWS environment. 
+
+Here are options to get started:
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                AWS solution
+            </th>
+            <th>
+                New Relic integration method
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                CloudWatch metric streams
+            </td>
+            <td>
+                [Set up Cloudwatch Metric Streams integration]() (Recommended)
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Amazon EC2 instances
+            </td>
+            <td>
+                [Install the infrastructure agent with CloudFormation template]()
+            </td>
+        </tr>
+        <tr>
+            <td>
+                AWS GovCloud (US)
+            </td>
+            <td>
+                [Connect your Amazon account to New Relic]()
+            </td>
+        </tr>
+        <tr>
+            <td>
+                AWS System Manager Distributor
+            </td>
+            <td>
+                [Install the infrastructure agent on AWS System Manager Distributor]()
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Amazon Elastic Kubernetes Service (Amazon EKS)
+            </td>
+            <td>
+                [Add New Relic to a custom resource definition (CRD) file]()
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Connect to the AWS Polling API
+            </td>
+            <td>
+                [Create new roles and policies to poll and send metric data to New Relic]()
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## View your AWS data [#view-aws-data]
+
+Once installed and configured, you can access data from your AWS services directly in the New Relic platform. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Amazon Web Services**</DNT>. You can start building queries from <DNT>**Metrics & Events**</DNT> as well, or by interacting with our query tool anchored at the bottom of your browser.
```

Comment:
@akristen The end of the last sentence is a little wobbly, I think: "or by interacting with our query tool anchored at the bottom of your browser." Maybe split this out into another sentence? There's also a keyboard shortcut that might be easier to mention: CTRL+SHIFT+O  Also, a screenshot might be helpful here. 

It may not be easy to get this, but it would also be great to include a sample, simple query for AWS content, if we can get one.